### PR TITLE
Docs: Added Universe

### PR DESCRIPTION
**Pull request by LinuxGamer**
Added Universe, an app to do anything and everything.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -176,3 +176,12 @@ Issues for first timers: https://github.com/TechSquidTV/Shoutify/contribute
 Built on: Go
 
 https://github.com/KyleTryon/Feed-Monitor
+
+## LinuxGamer/Universe
+
+> An open source app to do anything and everything.
```
Comment (ykdojo) [inline README.md:182]: I see what you're trying to do, but I think this description is too broad.

Could you phrase it in a more specific way?

**Review by ykdojo — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (LinuxGamer) [reply README.md]: How does it look now?

**Review by ykdojo — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ykdojo) [reply README.md]: looks good! thank you

File: README.md [hunk 2]
```diff
@@ -176,3 +176,12 @@ Issues for first timers: https://github.com/TechSquidTV/Shoutify/contribute
 Built on: Go
 
 https://github.com/KyleTryon/Feed-Monitor
+
```
Comment (ykdojo) [inline README.md:179]:
```suggestion

---

```

File: README.md [hunk 3]
```diff
@@ -176,3 +176,14 @@ Issues for first timers: https://github.com/TechSquidTV/Shoutify/contribute
...
+---
+
+## LinuxGamer/Universe
+
+> An open source app to do anything and everything. From a simple app launcher, to a terminal emulator, to a file manager, even to a web browser (all built in).
```
Comment (ykdojo) [inline README.md:184]:
```suggestion
> An open source app to do many things - from a simple app launcher, to a terminal emulator, to a file manager, and even to a web browser (all built-in).
```

**Comment by ykdojo** [comment]
@LinuxGamer I changed some stuff - can you check to see if it's ok?

**Comment by LinuxGamer** [comment]
That looks fine, thanks 👍🏼

**Pull request merged**